src/services/passwordGeneration.service.ts

```diff
@@ -256,6 +268,16 @@ export class PasswordGenerationService implements PasswordGenerationServiceAbstr
         return result;
     }
 
+    private capitalizeFirstLetter(str: string) {
```

Comment:
Just call it `capitalize`, which already means uppercasing the first letter in a string.